torch/csrc/deploy/CMakeLists.txt

```diff
@@ -1,3 +1,27 @@
 set(DEPLOY_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
 
 add_subdirectory(interpreter)
+
+add_custom_command(
+  OUTPUT libtorch_deployinterpreter.o
+  COMMAND cp $<TARGET_FILE:torch_deployinterpreter> .
```

Comment:
is this cp and the following rm necessary? or can you just pass this $<> to ld?

Reply:
the name of the symbol is determined by the filename :( 

I can't find a flag to set otherwise, so I have to do this silly copy.